src/services/formatting/ruleOperation.ts

```diff
@@ -1,4 +1,5 @@
 ///<reference path='references.ts' />
+/* tslint:disable:no-null-keyword */

```

Comment:
Never mind, I got to the more extensive uses of null below. Still might be worth asking around to see if you can learn the reason for null usage here and nowhere else.